libs/checkpoint/langgraph/checkpoint/memory.py

```diff
@@ -70,10 +71,11 @@ def get_tuple(self, config: RunnableConfig) -> Optional[CheckpointTuple]:
             Optional[CheckpointTuple]: The retrieved checkpoint tuple, or None if no matching checkpoint was found.
         """
         thread_id = config["configurable"]["thread_id"]
-        if ts := config["configurable"].get("thread_ts"):
-            if saved := self.storage[thread_id].get(ts):
-                checkpoint, metadata, parent_ts = saved
-                writes = self.writes[(thread_id, ts)]
+        checkpoint_ns = config["configurable"].get("checkpoint_ns", "")
+        if checkpoint_id := config["configurable"].get("checkpoint_id"):
```

Comment:
we can make this change backwards compat by reading thread_ts if checkpoint_id not found, we probably should? Even though we are making breaking changes always good to minimize them as much as we can

Reply:
sounds good to me